user-docs/documentation/docs/how-to-guides/1-use-serverless-contracts/1-api-gateway.md

```diff
@@ -104,11 +105,11 @@ In the `config.ts` file of our lambda, in the `events` section, we need to use t
 export default {
   environment: {},
   handler: getHandlerPath(__dirname),
-  events: [getTrigger(myContract)],
+  events: [getTrigger(myContract, {})],
```

Comment:
Is there no way to have the type inference without having to add the parameter ?
We can maybe ask @ThomasAribart, the TS 🧙

Reply:
I don't know unfortunately 🤔

Reply:
@adriencaccia the TS 🧙🏻 has helped me and the argument is no longer required for unauthenticated contracts! Many thanks @ThomasAribart 💪🏻